PutJSONTogether/app.js

```diff
@@ -0,0 +1,70 @@
+var fs = require("fs");
+console.log("\n *START* \n");
+var content = "artists.json";
+
+var musicians = [];
+var newObj = {};
+
+function convertToInformation(content) {
+    fs.readFile(content, 'utf8', function readFileCallback(err, data){
+        if (err){
+            console.log(err);
```

Comment:
If you add a return here, you can save on the else block and have an indentation less.